tests/integ/test_huggingface.py

```diff
@@ -158,7 +158,7 @@ def test_huggingface_inference(
     huggingface_pytorch_latest_inference_py_version,
 ):
     env = {
-        "HF_MODEL_ID": "sshleifer/tiny-distilbert-base-uncased-finetuned-sst-2-english",
+        "HF_MODEL_ID": "philschmid/tiny-distilbert-classification",
```

Comment:
@philschmid Can we please give this a generic name to this model?

Reply:
Not sure what you mean by that. That is not a model which will be created it is stored on the hf.co/models hub and used to run tests. I changed it because to a model which we can control. https://huggingface.co/philschmid/tiny-distilbert-classification.